test/merchant_test.rb

```diff
@@ -50,10 +52,46 @@ def test_favorite_customer_returns_customer_with_most_successful_transactions
     assert_equal "Joey", result.first_name
   end
 
+  def test_customers_with_pending_invoices_returns_collection_of_customers_with_unpaid_invoices
+    load_data
+    result = @se.merchant_repository.merchants[2].customers_with_pending_invoices
+    assert_equal 0, result.count
+  end
+
   private
 
   def fixtures_directory
     File.expand_path('../../data/fixtures', __FILE__)
   end
 
+  def load_data
+    repo_data = {
+      merchants: [
+       { id: 10, name: "merchant_1" },
+       { id: 20, name: "merchant_2" },
+       { id: 30, name: "merchant_3" }
+     ],
+
+       invoices: [
+         { id: 1, merchant_id: 10, customer_id: 100, created_at: "2012-03-24" },
+         { id: 2, merchant_id: 20, customer_id: 100, created_at: "2012-03-24" },
+         { id: 3, merchant_id: 30, customer_id: 200, created_at: "2012-03-24" },
+         { id: 4, merchant_id: 30, customer_id: 300, created_at: "2012-03-24" }
+       ],
+       transactions: [
+       { id: 6, invoice_id: 1, result: "failed" },
+       { id: 7, invoice_id: 2, result: "failed" },
+       { id: 8, invoice_id: 3, result: "success" },
+       { id: 9, invoice_id: 4, result: "failed" },
+       { id: 0, invoice_id: 4, result: "success" }
+     ],
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.